### PR TITLE
feat(sdk-coin-osmo): osmosis sdk init

### PR DESCRIPTION
**Pull request by DinshawKothari**
Ticket: BG-78362

This PR includes the following changes -
  feat(statics): add OSMO coin config
  feat(sdk-coin-osmo): create osmo coin module
  feat(sdk-coin-osmo): add keyPair management to osmosis
  feat(sdk-coin-osmo): skeleton for transaction building and account-lib OSMO

### CODE REVIEW

**Comment by Vijay-Jagannathan** [comment]
I think it would be easier to review/follow if we could break up this PR into independent chunks

**Comment by DinshawKothari** [comment]
> I think it would be easier to review/follow if we could break up this PR into independent chunks

@Vijay-Jagannathan It will be difficult I guess to break it down in different PRs. But I have another PR with same changes, I have closed it due to the branch name change. May be you can take a look at that, it has logic based segregated commits.
https://github.com/BitGo/BitGoJS/pull/3566

File: CODEOWNERS
Comment (dpkjnr) [inline CODEOWNERS:52]: shouldn't it be ethalt now? 🤔

**Review by LeoKotschenreuther — COMMENTED** [review]
This is too much code at once to thoroughly review. Please get a review from somebody else or break it up.

If you submit the codeowners changes in a separate PR and get that PR merged then your following PRs will potentially need less reviews. I think GitHub determines who needs to review based on the Codeowners file in master and not based on the one in your PR.

**Review by sachushaji — COMMENTED** [review]
Lets do smaller PRs for this

File: modules/account-lib/package.json
Comment (mmcshinsky-bitgo) [inline modules/account-lib/package.json:43]: Do we need this for other services using account-lib? If not, let's remove this.

Comment (mmcshinsky-bitgo) [reply modules/account-lib/package.json:43]: If needed, also needs tsconfig update.

File: modules/sdk-coin-osmo/test/unit/transaction.ts
Comment (mmcshinsky-bitgo) [inline modules/sdk-coin-osmo/test/unit/transaction.ts:126]: Can we remove these or do we need these?

**Review by dpkjnr — APPROVED** [review]

**Pull request merged**
